src/imex.rs

```diff
@@ -494,10 +489,20 @@ async fn import_backup(context: &Context, backup_to_import: impl AsRef<Path>) ->
     );
 
     let backup_file = File::open(backup_to_import).await?;
+    let file_size = backup_file.metadata().await?.len();
     let archive = Archive::new(backup_file);
+
     let mut entries = archive.entries()?;
     while let Some(file) = entries.next().await {
         let f = &mut file?;
+
+        let current_pos = f.raw_file_position();
+        let progress = 1000 * current_pos / file_size;
+        if progress > 10 && progress < 1000 {
```

Comment:
I think it should be rescaled to the range 10..1000 rather than being stuck until progress reaches 10.

Reply:
A progress of 10 means 1%, which is the smallest progress most UIs can show. So, for a progress on such a small scale, the progress bar shown to the user will be stuck anyway.

That being said, of course I can change it if I didn't convince you :)